dev/Expander/Expander.xaml

```diff
@@ -20,17 +20,15 @@
                Without this override, the narrator user that focuses the expander on a touch screen will see that pressing "Tab"
                doesn't work how they would expect.-->
         <Setter Property="IsTabStop" Value="False"/>
-        <Setter Property="Background" Value="{ThemeResource ExpanderBackground}" />
+        <Setter Property="Background" Value="{ThemeResource ExpanderContentBackground}" />
```

Comment:
I expect some push back on the names here. I'm setting them to a resources named something like `ExpanderContentBackground` instead of more standardized `ExpanderBackground`. After reviewing the resources for expander I realized the best approach is to keep the naming completely clear on which area of the expander the resource applies to (header vs content). This is for both maintainability and for application developers who might need to customize these properties.

Reply:
Note that the code before this PR also used the same naming convention in one case `<Setter Property="Padding" Value="{StaticResource ExpanderHeaderPadding}" />`.

Reply:
I think your proposed names makes sense.

---
In reply to: [593535877](https://github.com/microsoft/microsoft-ui-xaml/pull/4394#discussion_r593535877) [](ancestors = 593535877)